### PR TITLE
Add progress bar for audio and editable buttons.

### PR DESCRIPTION
**Pull request by adam-j-turner**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -40,12 +58,27 @@ class Container extends React.Component {
...
       <div>
-        <Line percent={this.state.progress} />
-        <button onClick={this.playSound}>
-          Sound 1
-        </button>
+        <div className='header'>
+          <Line className='progressBar' percent={this.state.progress} />
+          <div className='editModeBar'>
+            <h1>Edit Mode: </h1>
+            <ToggleButtonGroup name="editMode" type="radio" onChange={this.changeEditMode}>
```
Comment (CollinEMac) [inline src/App.js]: Edit Mode 'OFF' button should be pre-checked. You can do this by adding `defaultValue={false}` to the ToggleButtonGroup.

File: src/App.js [hunk 2]
```diff
@@ -26,6 +41,9 @@ class Container extends React.Component {
   }
 
   playSound() {
```
Comment (CollinEMac) [inline src/App.js]: This method should reset the progress to 0. If you click double click the soundbutton, the progress bar will infinitely loop. The progress bar should start over on each click.

File: src/App.js [hunk 3]
```diff
@@ -40,12 +58,27 @@ class Container extends React.Component {
   }
 
   render() {
+    var layout = [
```
Comment (CollinEMac) [inline src/App.js]: I think this should be `const`. It should probably be defined somewhere else too.

File: src/App.js (hunk 1 above)
Comment (adam-j-turner) [reply src/App.js]: Fixed

File: src/App.js (hunk 2 above)
Comment (adam-j-turner) [reply src/App.js]: Fixed

File: src/App.js (hunk 3 above)
Comment (adam-j-turner) [reply src/App.js]: Updated. Will leave it there until we find a better place for it.

**Pull request merged**